api/v1alpha1/serverless_helpers.go

```diff
@@ -22,10 +22,11 @@ func (s *Serverless) IsConditionTrue(conditionType ConditionType) bool {
 }
 
 const (
-	DefaultEnableInternal  = false
-	DefaultRegistryAddress = "k3d-kyma-registry:5000"
-	DefaultServerAddress   = "k3d-kyma-registry:5000"
-	FeatureDisabled        = "disabled"
+	DefaultEnableInternal   = false
+	DefaultRegistryAddress  = "k3d-kyma-registry:5000"
+	DefaultServerAddress    = "k3d-kyma-registry:5000"
+	FeatureDisabled         = ""
```

Comment:
We will have empty string in case of disabled tracing or eventing. Is it on purpose?

Reply:
yes, that is the purpose. 
logic in the runtimes reacts to any non empty collector endpoint
https://github.com/kyma-project/kyma/blob/main/components/function-runtimes/nodejs/lib/tracer.js#L52

Reply:
we will show dash.. I think its good enough
![Screenshot 2023-08-08 at 12 39 32](https://github.com/kyma-project/serverless-manager/assets/6783567/f8295567-0c13-4a33-a2ab-97d4892346b9)

Reply:
But in CR status, not in busola